src/crate/crash/command.py

```diff
@@ -369,6 +374,38 @@ def _show_tables(self, *args):
                       where schema_name not in ('sys','information_schema')""")
 
     @noargs_command
+    def check(self, *args):
+        success = self._execute("""select description as "Failed checks"
+                                   from sys.checks
+                                   where severity > 5""")
+        self.exit_code = self.exit_code or int(not success)
+        if not success:
+            return False
+        cur = self.cursor
+        print_vars = {
+            's': 'S'[cur.rowcount == 1:],
+            'rowcount': cur.rowcount
+        }
+        checks = cur.fetchall()
+        if len(checks):
+            self.pprint(checks, [c[0] for c in cur.description])
+            tmpl = '{rowcount} CHECK{s} FAILED'
+            self.logger.critical(tmpl.format(**print_vars))
+        else:
+            tmpl = 'CLUSTER CHECK OK'
+            self.logger.info(tmpl.format(**print_vars))
+        return True
+
+    @noargs_command
+    def _check(self, *args):
+        """ print failed cluster checks """
+        if self.connection.lowest_server_version >= CHECK_MIN_VERSION:
+            self.check()
+        else:
+            tmpl = '\nCrate {version} version does not support the cluster "check" command'
```

Comment:
also check if the command was called via `\check` or on initial connect,
there should not be a warning on initial connect and Crate does not support cluster checks